x/profiles/client/utils/utils.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // NewChainLinkJSON allows to build a new ChainLinkJSON instance
-// nolint: interfacer
+//nolint:interfacer
```

Comment:
This and similar changes are to make sure the lint process is compatible with the latest golint 1.48 version